src/cmap/command_monitoring_events.ts

```diff
@@ -139,7 +165,8 @@ export class CommandFailedEvent {
     connection: Connection,
     command: WriteProtocolMessageType,
     error: Error | Document,
-    started: number
+    started: number,
+    serverConnectionId?: bigint | undefined
```

Comment:
```suggestion
    serverConnectionId: bigint | null
```

`null !== undefined` if we want to fix this to be defaulted to null then it must only support bigint or null and not allow optionality.